### ISSUE TITLE
New Installation Error - Payment Gone

### ISSUE THREAD

**Issue by wilandtrading** [EvernodeXRPL/evernode-host#286]
Hi!
I have a new installation error which makes me have to create this new issue.
I have already an open issue where a payment has gone, and now its a new installation.
The previous issue hasnt been answered/resolved yet, and now I have yet another attempt to install where my payment is gone.

I have invested a fair bit of time and money in this project so I hope these matters can be resolved quickly.

This new installation error is regarding host address rPs5Ax7rXMeBKmprVYQFsPXCRiKrJB4d8B
When I try to do a new installation (transfer) it says host is not registered. When I then try to do a install (with 'n' as input if its a fresh install) it at some point asks for the host address, after I input the host address it then says it needs minimum 500 EVR and current balance is 10 EVR.

The problem is that the 500 EVR has already been paid.

The hash key for the transaction is here:
250101E6C20A9A252322A74A7FFD4726CA4BF1806849DB00865BBD4B70272629

Please help me with this asap.

**Comment by wilandtrading** [comment]
Same problem with host address: rh9mGYj3xk1taV5n11RwzXb4ZDy1kpot8m

The hash key for the transaction is here:
11DE0C6A5306BA3A72E1034504C3BFF128D60D152343B7B4DCA3473A562E71FF

So its now two installations where if I run the transfer it says its not registered, and when I then do the install it says it is not enough EVR on these host addresses, but as you can see its already been paid.

How can I get these refunded and re-installed?

**Comment by wilandtrading** [comment]
Hi again,
I managed to get this working by logging in to xahauexplorer.com with the xaman account which is attached to the host address. Then I find the last transaction that stopped and on there I click on the NFT hash. This gave me the opportunity to buy this NFT for a very small amount of XAH. 
Once you have bought the NFT (of the last transaction that failed) you run the transfer script, followed by the install script.

Closing the issue.
